.travis.yml

```diff
@@ -26,4 +26,14 @@ script:
   - lua -lluacov spec.lua
 
 after_success:
-  - luacov-coveralls -e $TRAVIS_BUILD_DIR/lua_install
\ No newline at end of file
+  - luacov-coveralls -e $TRAVIS_BUILD_DIR/lua_install
+  - mkdocs build
+
+deploy:
+  provider: pages
+  skip-cleanup: true
+  github-token: $GITHUB_TOKEN
+  keep-history: true
+  local_dir: site
+  on:
+    branch: master
```

Comment:
I don't *think* so. Travis will not pass the token to builds on forks - it will only pass the token to PRs that originate from this repository:

> Similarly, we do not provide [environment variables] to untrusted builds, triggered by pull requests from another repository.

So long as we're careful to vet any changes to `.travis.yml`, we should be okay in that regard. This is only set to deploy to `gh-pages` for commits on the `master` branch, so we shouldn't have to worry about someone hijacking the docs with a malicious PR.

I'm not sure that `gh-deploy` would be better than letting Travis deploy it. `gh-deploy` will create the commit in the `gh-pages` branch, but it won't push it, and pushing it could be...problematic.

Reply:
If MkDocs' `gh-deploy` command is equivalent to wiping out `site`, building a new `site`, and then committing + pushing to the `gh-pages` branch, then it's reasonable to not use it. Travis probably needs to do some trickery to push safely back to Git!